### PR TITLE
One Month: Add a status report for first month of tBTC

### PR DESCRIPTION
**Pull request by Shadowfiend**
This details various issues, actions, and statistics of the technical
side of tBTC's first month.

### CODE REVIEW

File: docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+  Bitcoin transaction exists on the Bitcoin chain and has been confirmed a
+  certain number of times. Without this component, funding and redemption
+  proofs cannot be submitted to tBTC, leading to the system's
+  deposit/redemption flow stalling out. Bitcoin can still flow, and any
+  unproven funds are still recoverable as with most other states the system
+  can enter, but obviously this is not ideal. The underlying causes for this
+  particular issue are still under investigation---a balance check
+  notification and a page for the insufficient funds issue on the maintainer
+  both failed to fire. Fortunately, the team at
+  strudel.finance[https://strudel.finance] noticed this issue shortly after
```
Comment (pdyraga) [inline docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc:245]: `strudel.finance[https://strudel.finance]` link looks wrong when rendered.

File: docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+  coordination in the default case.
+* **Relay maintainer running out of gas**. The cross-chain nature of tBTC
+  requires an SPV relay, which enables Ethereum contracts to confirm that a
+  Bitcoin transaction exists on the Bitcoin chain and has been confirmed a
+  certain number of times. Without this component, funding and redemption
+  proofs cannot be submitted to tBTC, leading to the system's
+  deposit/redemption flow stalling out. Bitcoin can still flow, and any
+  unproven funds are still recoverable as with most other states the system
+  can enter, but obviously this is not ideal. The underlying causes for this
+  particular issue are still under investigation---a balance check
```
Comment (pdyraga) [inline docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc:242]: `---` is rendered as just ---. Is it what we wanted to achieve?

**Comment by pdyraga** [comment]
One question left and I am good to 🚢 .

File: docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc (hunk 2 above)
Comment (Shadowfiend) [reply docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc:242]: No, but it'll do.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**